webpack.config.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
     libraryTarget: 'commonjs2',
   },
   module: {
-    preLoaders: preLoaders,
+    preLoaders,
```

Comment:
Does this work? I don't know if the webpack config file is run through babel. It may need to be ignored by eslint for this rule.